third_party/freetype2/autogen.sh

```diff
@@ -162,4 +172,29 @@ cd ../..
 
 chmod +x ./configure
 
+# Copy all necessary 'dlg' files.
+copy_submodule_files ()
+{
+  echo "Copying files from \`subprojects/dlg' to \`src/dlg' and \`include/dlg'"
+  mkdir include/dlg 2> /dev/null
+  cp $DLG_INC_DIR/output.h include/dlg
+  cp $DLG_INC_DIR/dlg.h include/dlg
+  cp $DLG_SRC_DIR/* src/dlg
+}
+
+if test -e ".git"; then
+  DLG_INC_DIR=subprojects/dlg/include/dlg
+  DLG_SRC_DIR=subprojects/dlg/src/dlg
+
+  if test -d "$DLG_INC_DIR"; then
+    :
+  else
+    echo "Checking out submodule in \`subprojects/dlg':"
```

Comment:
I found all workspace on win32 runners have this code and I think it's causing git checkout problem.